recipes/sqlobject/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "SQLObject" %}
+{% set version = "3.5.0" %}
+{% set sha256 = "5d7615c23f0ed90094b492859239cb4d40d2ac8c3fa52bc7175cf417f8393509" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - formencode >=1.3.1
```

Comment:
I think only `python` and `setuptools` should be `build` requirements since you're using `--single-version-externally-managed`